salt/utils/pyobjects.py

```diff
@@ -332,6 +330,23 @@ def __set_attributes__(cls):
 
                 match_groups[grain][match][name] = filt.__dict__[name]
 
+        if hasattr(cls, 'priority'):
```

Comment:
Also, I would appreciate your feedback on the documentation I added to salt/renderers/pyobjects.py. I want to make sure that what I added is accurate.